api/client/contextdialer.go

```diff
@@ -195,7 +195,8 @@ func sshConnect(ctx context.Context, conn net.Conn, ssh ssh.ClientConfig, dialTi
 	// Build a net.Conn over the tunnel. Make this an exclusive connection:
 	// close the net.Conn as well as the channel upon close.
 	conn, _, err = sshutils.ConnectProxyTransport(sconn.Conn, &sshutils.DialReq{
-		Address: constants.RemoteAuthServer,
+		Address:       constants.RemoteAuthServer,
+		ClientSrcAddr: conn.RemoteAddr(),
```

Comment:
Why are we specifying a `ClientSrcAddr` here? And why is it the `RemoteAddr` for the connection to the proxy itself?

Reply:
Leftover from early draft, fixed.